### PR TITLE
Add performance-tests skeleton and example test

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/config/TopologyConfig.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* Copyright 2018 Telstra Open Source
```
Comment (andriidovhan) [inline services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/config/TopologyConfig.java]: 2018 -> 2019

File: services/src/performance-tests/pom.xml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                                    </includes>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+</project>
```
Comment (andriidovhan) [inline services/src/performance-tests/pom.xml]: Please, add a new line at the end of the file.

File: services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/healthcheck/HealthCheckExtension.groovy [hunk 3]
```diff
@@ -1,16 +1,18 @@
 package org.openkilda.functionaltests.extension.healthcheck
 
+import org.openkilda.functionaltests.extension.FeatureOrderExtension
```
Comment (ylobankov) [inline services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/healthcheck/HealthCheckExtension.groovy]: To be consistent with `perftests` it should be named `functests`. Or we can use `performancetests`, `functionaltests`.

Comment (rtretyak) [reply services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/extension/healthcheck/HealthCheckExtension.groovy]: changed to performancetests

**Pull request merged**